### PR TITLE
Remove references to the `LanguageServerProtocol` modules in that module itself

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test

File: Sources/LanguageServerProtocol/SupportTypes/ClientCapabilities.swift [hunk 1]
```diff
@@ -86,26 +86,26 @@ public struct WorkspaceClientCapabilities: Hashable, Codable, Sendable {
...
 
       /// The symbol kind values that the client can support.
       ///
       /// If not specified, the client support only the kinds from `File` to `Array` from LSP 1.
       ///
       /// If specified, the client *also* guarantees that it will handle unknown kinds gracefully.
-      public var valueSet: [LanguageServerProtocol.SymbolKind]? = nil
+      public var valueSet: [SymbolKindValueSet]? = nil
 
-      public init(valueSet: [LanguageServerProtocol.SymbolKind]? = nil) {
```
Comment (hamishknight) [inline Sources/LanguageServerProtocol/SupportTypes/ClientCapabilities.swift:98]: These renames don't seem right, these should be the top-level `SymbolKind`, right?

Comment (ahoppen) [reply Sources/LanguageServerProtocol/SupportTypes/ClientCapabilities.swift:98]: You are right.

**Review by hamishknight — APPROVED** [review]

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**